client/src/components/Common/selectBox/index.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import 'antd/dist/antd.css';
+import './index.css';
+import { Select, ConfigProvider } from 'antd';
+import PropTypes from 'prop-types';
+
+const { Option } = Select;
+
+const SelectBox = ({
```

Comment:
why you pushed the select box with this pull request, isn't it has one alone ??? I think in your sider you don't have to add a select box for the **name** as you did in Figma, just add it in box or div by appropriate way .